src/lib/Server/Controller/Role.php

```diff
@@ -377,7 +383,7 @@ public function loadPolicy($roleId, $policyId, Request $request)
     /**
      * Adds a policy to role.
      *
-     * @param $roleId int ID of a role or a role draft
+     * @param $roleId int ID of a role draft
```

Comment:
```suggestion
     * @param int $roleId ID of a role draft
```
while at it, can we introduce `int` to `addPolicy` 1st argument itself or is it too risky?